### PR TITLE
many new attributes

### PR DESCRIPTION
**Pull request by travisghansen**

### CODE REVIEW

**Comment by travisghansen** [comment]
Still going to add the `_time` logic so no merge just yet..

**Comment by travisghansen** [comment]
@vinteo ok, I think this is ready unless we want to do `None` for the various durations...I'm guessing not.

File: custom_components/opensprinkler/__init__.py [hunk 1]
```diff
@@ -119,32 +119,94 @@ def _get_state(self):
...
             "master_station_1",
+            "master_station_1_time_on_adjustment",
+            "master_station_1_time_off_adjustment",
             "master_station_2",
+            "master_station_2_time_on_adjustment",
+            "master_station_2_time_off_adjustment",
             "rain_delay_active",
             "rain_delay_stop_time",
-            "rain_sensor_enabled",
+            "rain_sensor_active",
```
Comment (vinteo) [inline custom_components/opensprinkler/__init__.py:155]: could have sworn I changed this previously? lol

Comment (vinteo) [reply custom_components/opensprinkler/__init__.py:155]: oh wait it there two lines above, remove this one

**Pull request merged**